diff --git a/node_modules/@supabase/realtime-js/dist/module/RealtimeChannel.d.ts b/node_modules/@supabase/realtime-js/dist/module/RealtimeChannel.d.ts
index cad5764..a44271d 100644
--- a/node_modules/@supabase/realtime-js/dist/module/RealtimeChannel.d.ts
+++ b/node_modules/@supabase/realtime-js/dist/module/RealtimeChannel.d.ts
@@ -1,9 +1,9 @@
-import { CHANNEL_STATES } from './lib/constants';
-import Push from './lib/push';
-import type RealtimeClient from './RealtimeClient';
-import Timer from './lib/timer';
-import RealtimePresence, { REALTIME_PRESENCE_LISTEN_EVENTS } from './RealtimePresence';
-import type { RealtimePresenceJoinPayload, RealtimePresenceLeavePayload, RealtimePresenceState } from './RealtimePresence';
+import { CHANNEL_STATES } from '@supabase/realtime-js/src/lib/constants';
+import Push from '@supabase/realtime-js/src/lib/push';
+import type RealtimeClient from '@supabase/realtime-js/src/RealtimeClient';
+import Timer from '@supabase/realtime-js/src/lib/timer';
+import RealtimePresence, { REALTIME_PRESENCE_LISTEN_EVENTS } from '@supabase/realtime-js/src/RealtimePresence';
+import type { RealtimePresenceJoinPayload, RealtimePresenceLeavePayload, RealtimePresenceState } from '@supabase/realtime-js/src/RealtimePresence';
 type ReplayOption = {
     since: number;
     limit?: number;
diff --git a/node_modules/@supabase/realtime-js/dist/module/RealtimeClient.d.ts b/node_modules/@supabase/realtime-js/dist/module/RealtimeClient.d.ts
index f119877..dc55247 100644
--- a/node_modules/@supabase/realtime-js/dist/module/RealtimeClient.d.ts
+++ b/node_modules/@supabase/realtime-js/dist/module/RealtimeClient.d.ts
@@ -1,9 +1,9 @@
-import { WebSocketLike } from './lib/websocket-factory';
-import { CONNECTION_STATE } from './lib/constants';
-import Serializer from './lib/serializer';
-import Timer from './lib/timer';
-import RealtimeChannel from './RealtimeChannel';
-import type { RealtimeChannelOptions } from './RealtimeChannel';
+import { WebSocketLike } from '@supabase/realtime-js/src/lib/websocket-factory';
+import { CONNECTION_STATE } from '@supabase/realtime-js/src/lib/constants';
+import Serializer from '@supabase/realtime-js/src/lib/serializer';
+import Timer from '@supabase/realtime-js/src/lib/timer';
+import RealtimeChannel from '@supabase/realtime-js/src/RealtimeChannel';
+import type { RealtimeChannelOptions } from '@supabase/realtime-js/src/RealtimeChannel';
 type Fetch = typeof fetch;
 export type Channel = {
     name: string;
diff --git a/node_modules/@supabase/realtime-js/dist/module/RealtimePresence.d.ts b/node_modules/@supabase/realtime-js/dist/module/RealtimePresence.d.ts
index 583618b..97f82d1 100644
--- a/node_modules/@supabase/realtime-js/dist/module/RealtimePresence.d.ts
+++ b/node_modules/@supabase/realtime-js/dist/module/RealtimePresence.d.ts
@@ -1,5 +1,5 @@
 import type { PresenceOpts, PresenceOnJoinCallback, PresenceOnLeaveCallback } from 'phoenix';
-import type RealtimeChannel from './RealtimeChannel';
+import type RealtimeChannel from '@supabase/realtime-js/src/RealtimeChannel';
 type Presence<T extends {
     [key: string]: any;
 } = {}> = {
diff --git a/node_modules/@supabase/realtime-js/dist/module/lib/push.d.ts b/node_modules/@supabase/realtime-js/dist/module/lib/push.d.ts
index 604c4f7..08f02d9 100644
--- a/node_modules/@supabase/realtime-js/dist/module/lib/push.d.ts
+++ b/node_modules/@supabase/realtime-js/dist/module/lib/push.d.ts
@@ -1,4 +1,4 @@
-import type RealtimeChannel from '../RealtimeChannel';
+import type RealtimeChannel from '@supabase/realtime-js/src/RealtimeChannel';
 export default class Push {
     channel: RealtimeChannel;
     event: string;
